src/guides/getting-started.md

```diff
@@ -573,6 +573,58 @@ That's it! Let's try it out with `cargo run --bin publish_post 1`.
 Published post Diesel demo
 ```
 
+Additionally let's implement a possibility of fetching a single post. We will display the post id with its title.
+Notice `optional()` call. This returns `Option<Post>` instead of throwing an error, which we can then use in our matching pattern.
+
+::: code-block
+
+[src/bin/get_post.rs](https://github.com/diesel-rs/diesel/blob/2.1.x/examples/postgres/getting_started_step_3/src/bin/get_post.rs)
+
+```rust
+use self::models::Post;
+use diesel::prelude::*;
+use diesel_demo_step_3_sqlite::*;
+use std::env::args;
+
+fn main() {
+    use self::schema::posts::dsl::{id, posts};
+
+    let post_id = args()
+        .nth(1)
+        .expect("get_post requires a post id")
+        .parse::<i32>()
+        .expect("Invalid ID");
+    
+    let connection = &mut establish_connection();
+
+    let post = connection
+        .transaction(|connection| {
+            let post = posts.find(id).select(Post::as_select())
```

Comment:
Maybe add an additional sentence to the text above to point to the API documentation of `QueryDsl` for additional methods? That should make it much easier to discover all the other existing methods.